CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
   * `-ingester.read-path-memory-utilization-limit`
 * [FEATURE] Ruler: Support filtering results from rule status endpoint by `file`, `rule_group` and `rule_name`. #5291
 * [FEATURE] Ingester: add experimental support for creating tokens by using spread minimizing strategy. This can be enabled with `-ingester.ring.token-generation-strategy: spread-minimizing` and `-ingester.ring.spread-minimizing-zones: <all available zones>`. In that case `-ingester.ring.tokens-file-path` must be empty. #5308 #5324
+* [FEATURE] Ingester: add experimental CLI flag `-ingester.ring.token_generation_can_join_check_enabled` for enabling token generator's CanJoin check. #5541
```

Comment:
```suggestion
* [FEATURE] Ingester: add experimental CLI flag `-ingester.ring.token-generation-can-join-check-enabled` for enabling token generator's CanJoin check. #5541
```

Reply:
Updated with the new CLI flag, `-ingester.ring.spread-minimizing-join-ring-in-order`